docs/permissions/limitation_reference.md

```diff
@@ -291,6 +291,25 @@ Object state.
 |------|------|------|
 |`<ObjectState_id>`|`<ObjectState_name>`|All valid Object state IDs can be set as value(s)|
 
+## OrderOwner Limitation
+
+A Limitation to specify whether the user can modify an order.
+
+|                 |                                                                  |
+|-----------------|------------------------------------------------------------------|
+| Identifier      | `OrderOwner`                                                     |
+| Value Class     | `Ibexa\OrderManagement\Security\Policy\Limitation\OrderOwner`    |
+| Type Class      | `Ibexa\OrderManagement\Security\Policy\Limitation\OrderOwnerType`| 
+| Criterion used  | n/a                                                              |
+| Role Limitation | no                                                               |
+
+### Possible values
+
+|Value|UI value|Description|
+|------|------|------|
+|"self"|"self"|Users can access only their own orders. |
+|"all"| none |Users can access all orders.|
```

Comment:
This isn't the case. We don't need that limitation as it's basically an equivalent of adding `Order/*` policy without any limitations.